firebase-vscode/src/workflow.ts

```diff
@@ -369,10 +373,54 @@ export function setupWorkflow(
       readAndSendFirebaseConfigs(broker);
       broker.send("notifyHostingFolderReady",
         { projectId, folderPath: currentOptions.cwd });
-      
+
       await fetchChannels();
     }
   }
+  broker.on(
+    "launchEmulators",
+    async ({ firebaseJson, emulatorUiSelections }) => {
+      emulatorsStart(firebaseJson, emulatorUiSelections).then(() => {
```

Comment:
Prefer await emualtorsStart() over promise chaining, especially sincd were already in an async function

Reply:
good point, fixed!